dashboard/app/controllers/pd/professional_learning_landing_controller.rb

```diff
@@ -9,20 +9,35 @@ def index
       return
     end
 
-    last_enrollment_with_pending_survey = Pd::Enrollment.filter_for_survey_completion(
+    enrollments_with_pending_surveys = Pd::Enrollment.filter_for_survey_completion(
       Pd::Enrollment.for_user(current_user).with_surveys,
       false
-    ).max_by {|e| e.workshop.ended_at}
+    )
+    last_enrollment_with_pending_survey = enrollments_with_pending_surveys.max_by {|e| e.workshop.ended_at}
 
     summarized_plc_enrollments = Plc::UserCourseEnrollment.where(user: current_user).map(&:summarize).sort_by do |enrollment|
       PLC_COURSE_ORDERING.index(enrollment[:courseName]) || PLC_COURSE_ORDERING.size
     end
 
+    enrolled_workshop_data = Pd::Enrollment.for_user(current_user).map do |enrollment|
+      workshop = enrollment.workshop
+      {
+        course_name: workshop.course_name,
```

Comment:
@bethanyaconnor are these the keys I'll use when calling data?

Reply:
yep!